charts/flagsmith/templates/secrets-api.yaml

```diff
@@ -7,7 +7,9 @@ metadata:
     app.kubernetes.io/component: api
 type: Opaque
 data:
+{{- if .Values.databaseExternal.url }}
```

Comment:
Similarly to https://github.com/Flagsmith/flagsmith-charts/pull/102/files#r987786252, I don't think this is right.